modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/UniqueCompositeKeyObjectValidationRuleEngineImpl.java

```diff
@@ -139,7 +139,9 @@ private Predicate _getPredicate(
 				Map<String, Object> objectFieldProperties =
 					(Map<String, Object>)entryValues.get(objectField.getName());
 
-				value = String.valueOf(objectFieldProperties.get("key"));
+				if (objectFieldProperties != null) {
```

Comment:
Hey @mateussgn , what happens if objectFieldProperties is null, is there a problem to pass value variable with null value to `ObjectEntrySearchUtil.getUniqueCompositeKeyObjectFieldPredicate((Column<?, Object>)column, objectField.getDBType(), value)`?